src/run.py

```diff
@@ -73,6 +76,39 @@ def copilot_qna(question, chat_completion_fn):
         "context": response["context"]
     }
 
+def evaluate_result(task_type,
+             data_mapping,
+             metrics_list,
+             dataset,
+             target=None,
+             evaluation_name="baseline-evaluation",
+             output_path="./evaluation_output"):
+    from azure.ai.generative.evaluate import evaluate
+
+    client = AIClient.from_config(DefaultAzureCredential())
```

Comment:
I think we may need to add the function `prepare_search_index` here because LangChain uses the MLIndex differently -- please delete the MLIndex folder from root directory (something like "[index-name]-mlindex") and then try again?